engine/src/main/resources/application.conf

```diff
@@ -78,86 +70,128 @@ call-caching {
-    // The following are strategies used to make those links.  They are ordered.  If one fails
-    // The next one is tried:
-    //
-    // hard-link: attempt to create a hard-link to the file
-    // copy: copy the file
-    // soft-link: create a symbolic link to the file
-    //
-    // NOTE: soft-link will be skipped for Docker jobs
-    localization: [
-      "hard-link", "soft-link", "copy"
-    ]
-  }
-
+system {
   // If 'true', a SIGINT will trigger Cromwell to attempt to abort all currently running jobs before exiting
-  abortJobsOnTerminate = false
+  abort-jobs-on-terminate = false
 }
 
-// If authenticating with Google (e.g. if backend is 'JES') you must supply a top level stanza 'google':
-// This defines how all requests to JES/GCS performed by Cromwell will be authenticated.
 google {
-  applicationName = "cromwell"
-
-  // cromwellAuthenticationScheme can take the values "service_account",
-  // "user_account", and "application_default". The first two are described
-  // in the stanzas below.
-  // Use "application_default" to use the default service account credentials.
-  // This is useful if you are running on a GCE VM and if you don't need
-  // user level access. See https://developers.google.com/identity/protocols/application-default-credentials
-  // for more info. No further information is needed.
-  cromwellAuthenticationScheme = "application_default"
-
-  // If cromwellAuthenticationScheme is "service_account"
-  serviceAuth {
-    // pemFile = ""
-    // serviceAccountId = ""
-  }
 
-  // If cromwellAuthenticationScheme is "user_account"
-  userAuth {
-    // user = ""
-    // secretsFile = ""
-    // dataStoreDir = ""
-  }
+  application-name = "cromwell"
+
+  auths = [
+    {
+      name = "application-default"
+      scheme = "application_default"
+    },
+    {
+      name = "user-via-refresh"
+      scheme = "refresh_token"
+      client-id = "secret_id"
+      client-secret = "secret_secret"
+    }
+  ]
+}
 
-  // [Optional]
-  // Can be defined to provide an alternative way for Cromwell to perform some runtime interactions with GCS.
-  // For instance data localization and delocalization in GoogleCloudStorage will be done on behalf of another entity (typically a user).
-  // This allows cromwell to localize data that wouldn't be accessible using whatever "cromwellAuthenticationScheme" has been defined.
-  // The currently only available scheme is refresh, which requires a refresh token to be passed upon every workflow submission (see "workflow options" in cromwell documentation).
-  // In that scheme, a client_id and client_secrets must be provided. They must be the ones that have been used to generate the above-mentioned refresh token.
-  // userAuthenticationScheme = "refresh"
-  // refreshTokenAuth = {
-  //   client_id = ""
-  //   client_secret = ""
-  // }
+backend {
+  default = "Local"
+  providers = [
+    {
+      name = "Local"
+      class = "cromwell.engine.backend.local.LocalBackend"
```

Comment:
I think we'll eventually want to name this differently - if we want to allow the optional initializationClass, executionClass, finalizationClass, ...